### PR TITLE
clarify which SDK path getSdkPath returns

### LINKED ISSUE #171: Rename path methods
Bad names just contributed to a major bug in CT4E. We should change

CloudSdk.getSdkPath --> CloudSdk.getPath
CloudSdk.getJavaAppEngineSdkPath --> CloudSdk.getJarDirectory

or perhaps something better

### PR DESCRIPTION
**Pull request by elharo**
@loosebazooka fix #171

### CODE REVIEW

File: src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java [hunk 1]
```diff
@@ -438,24 +437,26 @@ private static void logCommand(List<String> command) {
...
   }
 
   private Path getDevAppServerPath() {
-    return getSdkPath().resolve(DEV_APPSERVER_PY);
+    return getPath().resolve(DEV_APPSERVER_PY);
   }
 
-  public Path getJavaAppEngineSdkPath() {
-    return getSdkPath().resolve(JAVA_APPENGINE_SDK_PATH);
+  /** @return the directory containing JAR files bundled with the Cloud SDK */
```
Comment (briandealwis) [inline src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java]: I don't think this rename is helpful as there are a number of directories in the SDK that contain .jar files.

Comment (elharo) [reply src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java]: renamed

**Comment by loosebazooka** [comment]
Looks the merge changed the visibility of some methods. I think that needs to be updated.

**Comment by loosebazooka** [comment]
hrmm, I wonder what cobertuba's issue is, the kokoro tests appear to be failing because of checkstyle.
```
[WARNING] src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java:[441] (javadoc) SummaryJavadoc: Summary javadoc is missing.
[WARNING] src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java:[458] (javadoc) SummaryJavadoc: Summary javadoc is missing.
```

**Comment by elharo** [comment]
Cobertura may be getting tripped up by the lambdas.

**Pull request merged**